_data/tagging-status.yml

```diff
@@ -11008,13 +11022,13 @@
 - name: trimclip
   ctan-pkg: adjustbox
   type: package
-  status: unknown
+  status: partially-compatible
   included-in: [arxiv1]
   priority: 5
+  comments: "Depends on partially-compatible collectbox. `\clipbox` adds empty tags."
```

Comment:
the yaml validator is complaining here: we haven't been entirely consistent but I guess this should be `\\c` the backticks affect the display when this yaml is pulled in to the markdown but don't quote the backslash within the yaml.

Reply:
oh you fixed it:-)